52.md

```diff
@@ -203,9 +243,16 @@ The list of tags is as follows:
 }
 ```
 
-## Unsolved Limitations
+#### Private Event
+
+It's an _addressable event_ of `kind:32069`.
+Apart from the public event tags, there is one more tag:
+* `note` Note for the RSVP
+
+The corresponding gift wrap is of `kind:1052`
+The rumor for RSVP should contain the `a` tag for the encrypted RSVP event with value `<32069>:<viewer_pubkey>:<encrypted_rsvp_event_id>`
+The rumor for RSVP should also contain another `a` tag which will have value `<32069 or 31923>:<calendar event author pubkey>:<d-identifier of calendar event>`
```

Comment:
This is all very difficult to understand what you mean. Can you provide examples?